android_app/app/src/main/java/com/health/openscale/core/alarm/ReminderBootReceiver.java

```diff
@@ -27,6 +27,8 @@ public void onReceive(Context context, Intent intent)
     {
         if (intent.hasExtra(AlarmHandler.INTENT_EXTRA_ALARM)) handleAlarm(context);
 
+        if (intent.hasExtra(AlarmBackupHandler.INTENT_EXTRA_BACKUP_ALARM)) handleBackupAlarm(context);
```

Comment:
I've never gotten the reminder alarm to work. Do you have any idea why that could be?

Reply:
hmm not really for me it is working

Reply:
did you disable globally on your smartphone notifications?